internal/org/model/org.go

```diff
@@ -47,7 +46,8 @@ func (o *Org) GetPrimaryDomain() *OrgDomain {
 }
 
 func (o *Org) nameForDomain(iamDomain string) string {
```

Comment:
Can be deleted as only used in AddIAMDomain a few lines down and then can point to NewIAMDomainName.
A separate package utils can be deleted and included in internal/domain, as it is logic related to the domains.

Reply:
Absolutely

Reply:
Done!